devops/developer-api/redis/create_backup.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: "Create Backup"
+description: "This endpoint creates a backup for a Redis database."
+api: "POST https://api.upstash.com/v2/redis/create-backup/{id}"
+authMethod: "basic"
+---
+
+## URL Parameters
+
+<ParamField path="id" type="string" required>
+  The ID of the Redis database
+</ParamField>
+
+## Request Parameters
+
+<ParamField body="name" type="string" required>
+  Name of the backup
+</ParamField>
+
+## Response Parameters
+
+<ResponseField name="status" type="string">
```

Comment:
This endpoint only returns "OK" on successful request. When we say it like this, it looks like it returns {status: "BLABLA"} in the response body. Instead, we can just say "This endpoint returns 'OK' on successfull request".